modules/rtm/R/invert.custom.R

```diff
@@ -142,6 +151,11 @@ invert.custom <- function(observed, invert.options,
                       lower = param.mins, upper = param.maxs)
     inits <- fit$par
   }
+  if (!all(diag(init.Jump) > 0)) {
+    warning("Passed init.Jump matrix with zero values on diagonals. ", 
```

Comment:
you're not exactly checking only non-zeros here, although below you have another check for negative values as well.. maybe combine these into one check?